packages/replay-next/components/sources/log-point-panel/LogPointPanel.tsx

```diff
@@ -99,8 +99,8 @@ function PointPanel(props: ExternalProps & { focusRange: TimeStampedPointRange }
     hitPointsForLocationCache,
     client,
     { begin: focusRange.begin.point, end: focusRange.end.point },
-    pointForSuspense.location,
-    pointForSuspense.condition
+    pointForDefaultPriority.location,
+    pointForDefaultPriority.condition
```

Comment:
I've changed this to `pointForDefaultPriority` because since #9531 this component doesn't suspend anymore.